### DISCUSSION TITLE
Providing an ipa release option for ios

### DISCUSSION THREAD

**Discussion by arionukshini** [ppy/osu#18338] [Ideas] [answered]
Providing an ipa file for release for ios would be great because the testflight beta access gets filled fast and not all people can access it and i have seen some people compile the game into a ipa file but most of them are outdated or just scams. And if not providing a way on how to compile them ourselves would be great

**Comment by bdach** [comment, marked as answer]
not happening, see *several* past discussions and issues about this

https://github.com/ppy/osu/search?q=ipa&type=issues
https://github.com/ppy/osu/search?q=ipa&type=discussions

**Comment by peppy** [comment]
You can compile by opening the project and hitting the build button in your IDE of choice.
